### DISCUSSION TITLE
Headless CMS for use with Prisma

### DISCUSSION THREAD

**Discussion by devmanbr** [prisma/prisma#20605] [Q&A]
### Question

Does anyone know of an open-source Headless CMS that is flexible and can be used with Prisma?

I can't have the same flexibility in Strapi.

### Information about Prisma Schema, Client Queries and Environment (optional)

```js
// Add your schema.prisma and relevant query code
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @devmanbr 👋 

For a headless CMS that plays well with Prisma, you might want to check out KeystoneJS. It's a highly flexible framework for building GraphQL APIs, and offers its own ORM which can work in tandem with Prisma.

Here's the link to their website: https://www.keystonejs.com/

For usage examples with Prisma, I'd recommend browsing the KeystoneJS GitHub repository or their official documentation. 

Another one is [Directus](https://directus.io/) which is an open-source headless CMS with a highly flexible data modeling system. It allows you to define custom database schemas and connect to various SQL databases, which includes compatibility with Prisma. You can also check their Github repository or official documentation.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
